zarr/creation.py

```diff
@@ -19,32 +22,35 @@
     normalize_storage_path,
     normalize_store_arg,
 )
+from zarr._storage.store import StorageTransformer
+from zarr.sync import Synchronizer
+from zarr.types import ZARR_VERSION, DIMENSION_SEPARATOR, MEMORY_ORDER
 from zarr.util import normalize_dimension_separator
 
 
 def create(
-    shape,
-    chunks=True,
-    dtype=None,
+    shape: Union[int, Tuple[int, ...]],
+    chunks: bool = True,
+    dtype: Optional[Union[str, npt.DTypeLike]] = None,
```

Comment:
I think `str` is a subtype of `dtypeLike`: https://numpy.org/doc/stable/reference/typing.html#numpy.typing.DTypeLike

Reply:
(I should say, "the subset of `str` that can be coerced into numpy dtypes" is a subtype of `DtypeLike`)

Reply:
👍